### ISSUE TITLE
Financial Aid and SAAS Interface Re-Design

### ISSUE THREAD

**Issue by bryantal** [BCStudentSoftwareDevTeam/lsf#140]
**[Original report](https://bitbucket.org/laborstudents/lsf-flask/issue/140) by Hila Manalai (Bitbucket: [Hila Manalai](https://bitbucket.org/Hila Manalai), ).**

----------------------------------------

The Financial Aid and SAAS interface might need some small redesigning.   

1. The form should be placed in the center of the page. 
2. We will change the logo of the Labor Status Forms page, the same should be applied to this page. Changes will need to happen in financialAidBase.html. Or we would need to somehow integrate Base.html with the SAAS and Financial Aid interface. 
3. Any other changes you might think is needed in terms of design and accessibility. 

‌

**Comment by bryantal** [comment]
**Original comment by Hila Manalai (Bitbucket: [Hila Manalai](https://bitbucket.org/Hila Manalai), ).**

----------------------------------------

The first part of the issue is done. For the second part, I am waiting on the new logo to be merged into development to implement the same changes here.

**Comment by bryantal** [comment]
**Original comment by Bria Williams (Bitbucket: [BriaWilliams1202](https://bitbucket.org/BriaWilliams1202), ).**

----------------------------------------

Resolved in error

**Comment by bryantal** [comment]
**Original comment by Hila Manalai (Bitbucket: [Hila Manalai](https://bitbucket.org/Hila Manalai), ).**

----------------------------------------

PR has been merged.
